<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-F71TJG8P" failures="1" tests="1" name="FourTest" time="10.92" errors="0" timestamp="2022-09-01T16:30:49 IST">
  <testcase classname="Tests.FourTest" name="testFour" time="10.401">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Tests.FourTest.testFour(FourTest.java:27)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
at java.base/java.lang.Thread.run(Thread.java:834)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- testFour -->
</testsuite> <!-- FourTest -->
